app/models/user.rb

```diff
@@ -1,2 +1,9 @@
 class User < ApplicationRecord
+  has_many :comments, class_name: 'Comment', foreign_key: 'author_id'
+  has_many :likes, class_name: 'Like', foreign_key: 'author_id'
+  has_many :posts, class_name: 'Post', foreign_key: 'author_id'
```

Comment:
[OPTIONAL] Kindly remove the class_name present here because the purpose is to allow the naming of a class to be different from what Rails expects. And rails already knows that model names are to be singular